client/src/EventNew.js

```diff
@@ -56,7 +56,8 @@ export default function EventNew() {
       isRSVP: isRSVP
     }).then((response) => {
       // TODO: Redireccionar a todos los eventos
-      toast.success("Evento registrado correctamente")
+      toast.success("Evento registrado correctamente");
+      window.location.replace(`${url}/events`);
```

Comment:
Usa `history` esta libreria previene los reloads "lentos" del browser. En teoria React hace que la pagina sea de una sola vista por lo que los reloads son inecesarios.
```suggestion
      history.push('/events');
```

Reply:
Me agrada tu sugerencia y rpo eso se está haciendo en [este PR](https://github.com/APIStear/FETEC-News/pull/49)